presto-native-execution/presto_cpp/main/PrestoTask.cpp

```diff
@@ -129,6 +129,20 @@ static void addRuntimeMetricIfNotZero(
   }
 }
 
+bool shouldAggregateRuntimeMetric(const std::string& name) {
+  if (name == "dataSourceWallNanos" || name == "dataSourceLazyWallNanos" ||
+      name == "queuedWallNanos") {
```

Comment:
I'm okay with overall. It'll be better if we can make this enumeration in a const vector.

Reply:
+1